test/types/js.civet

```diff
@@ -188,9 +188,9 @@ describe "Types", ->
       testCase.js """
         plain function with @ shorthand
         ---
-        function (@: T) {}
+        function (@: T, x: X) {}
         ---
-        (function () {})
+        (function ( x) {})
       """
```

Comment:
Seems worth keeping the original (no-argument) case as well.
Hmm, though I guess the `this` (not `@`) case covers that.
Or we could have all four cases...